### PR TITLE
[Dynamic dashboard] Stats ViewModel unit tests

### PR DESCRIPTION
**Pull request by hichamboushaba**
Part of: #11219 

⚠️ Please don't merge until base branch is updated to `trunk`.

### Description
This simply adds unit tests for the Stats ViewModel.

### Testing instructions
Green CI is enough.

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/dashboard/stats/GetStatsTest.kt [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+import org.wordpress.android.fluxc.model.WCRevenueStatsModel
+import org.wordpress.android.fluxc.store.WCStatsStore.OrderStatsError
+import org.wordpress.android.fluxc.store.WCStatsStore.OrderStatsErrorType.GENERIC_ERROR
+import org.wordpress.android.fluxc.store.WCStatsStore.OrderStatsErrorType.PLUGIN_NOT_ACTIVE
+import java.util.Calendar
+import java.util.Date
+import java.util.Locale
+
+@ExperimentalCoroutinesApi
+class GetStatsTest : BaseUnitTest() {
```
Comment (hichamboushaba) [inline WooCommerce/src/test/kotlin/com/woocommerce/android/ui/dashboard/stats/GetStatsTest.kt:35]: This is an exact copy of the `GetStatsTest` in the old package.

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is not assigned to a milestone.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by 0nko — APPROVED** [review]
:shipit:

**Pull request merged**